src/OnlineSales/Interfaces/IContactService.cs

```diff
@@ -0,0 +1,17 @@
+﻿// <copyright file="IContactService.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using OnlineSales.Entities;
+
+namespace OnlineSales.Interfaces
+{
+    public interface IContactService
+    {
+        Task<Contact> AddContact(Contact contact);
+
+        Task<Contact> UpdateContact(Contact contact);
+
+        List<Contact> DomainMapperWithContacts(List<Contact> contacts);
```

Comment:
@WASandaruwan please rename to EnrichWithDomainId and make two separate overloads - one which accepts a list of contact and other which accept a single contact.